packages/shared/src/components/auth/AuthOptionsInner.tsx

```diff
@@ -223,12 +224,14 @@ function AuthOptionsInner({
       return displayToast(labels.auth.error.generic);
     },
   });
-  const onProfileSuccess = async (options: { redirect?: string } = {}) => {
+  const onProfileSuccess = async (
+    options: { redirect?: string; setSignBack?: boolean } = {},
+  ) => {
```

Comment:
I'm now using this `onProfileSuccess` after the signup with email. 
This was calling the `onSignBackLogin` that created redirection bugs within the funnel. Added an option to handle it.